web/src/pages/Route/components/DebugViews/DebugDrawView.tsx

```diff
@@ -51,6 +51,7 @@ const DebugDrawView: React.FC<RouteModule.DebugDrawProps> = (props) => {
   const [authForm] = Form.useForm();
   const [headerForm] = Form.useForm();
   const [responseCode, setResponseCode] = useState<string>();
```

Comment:
Hi @juzhiyuan 

I found the `setResponseCode` function,  actually deal with the response body, not response code,
so can I rename it?

Reply:
If it is, yes you can.